h/zis/services/auth.h

```diff
@@ -47,17 +47,18 @@ typedef struct AbendInfo_tag {
 
 #define ZIS_SAF_IDT_SERVICE_CURRENT_VERSION         1
 
-/* The #define for ZIS_AUTH_SERVICE_PARMLIST_OPTION_GENERATE_IDT has been deprecated, and will be removed
+/* The #define for ZIS_AUTH_SERVICE_SAFIDT_OPTION_GENERATE_IDT has been deprecated, and will be removed
    in the future.
   */
 typedef struct SafIdtService_tag{
   int safIdtServiceVersion;
   int safIdtLen;
-  #define ZIS_AUTH_SERVICE_PARMLIST_SAFIDT_LENGTH (2 * IDTA_IDT_BUFFER_LEN_MIN)
-  #define ZIS_AUTH_SERVICE_PARMLIST_OPTION_GENERATE_IDT 0x80
-  #define ZIS_AUTH_SERVICE_PARMLIST_OPTION_IDT_APPL 0x40
-  char safIdt[ZIS_AUTH_SERVICE_PARMLIST_SAFIDT_LENGTH];
+  #define ZIS_AUTH_SERVICE_SAFIDT_LENGTH (2 * IDTA_IDT_BUFFER_LEN_MIN)
+  #define ZIS_AUTH_SERVICE_SAFIDT_OPTION_GENERATE_IDT 0x80
+  #define ZIS_AUTH_SERVICE_SAFIDT_OPTION_IDT_APPL 0x40
+  char safIdt[ZIS_AUTH_SERVICE_SAFIDT_LENGTH];
   char applNullTerm[ZIS_AUTH_SERVICE_APPL_MAX_LENGTH + 1];
+  char options;
```

Comment:
Can you have it somewhere in the beginning of the struct, maybe after `safIdtLen`? 

And you can't just remove it from `AithServiceParmList` because it'll break the code. Replace it with `char _padding0[1];` instead.

Reply:
moved and replaced